plugins/woocommerce/src/Internal/StockNotifications/Enums/NotificationStatus.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Internal\StockNotifications\Enums;
+
+/**
+ * Enum class for all the notification statuses.
+ */
+final class NotificationStatus {
+	/**
+	 * The notification has been created but not yet confirmed.
```

Comment:
What do you think about expanding on these comments a little bit more?  What does confirmed mean?

Reply:
Absolutely! Expanded here https://github.com/woocommerce/woocommerce/pull/57062/commits/8cddd736c37e83efa6f5ead07ceffef0667c5403